src/utils/ValidationHelper.ts

```diff
@@ -94,6 +94,8 @@ export class ValidationHelper {
 			return `JWT validation/verification failed: Unable to retrieve redirect URI for client_id: ${requestBodyClientId}`;
 		} else if (expectedRedirectUri !== jwtPayload.redirect_uri) {
 			return `JWT validation/verification failed: Redirect uri ${jwtPayload.redirect_uri} does not match configuration uri ${expectedRedirectUri}`;
+		} else if (jwtPayload.context != null && expectedContext !== jwtPayload.context) {
```

Comment:
Sorry I'm a little confused about this, would you be able to elaborate? Are we saying where that we are rejecting any JWT where the context provided is not Constants.EXPECTED_CONTEXT (bank_account) ?

Reply:
Why is it considered an invalid request? I see in the mock we can attach anything we like

Reply:
I think IPV core may add anything in the context and hence the mock is doing that as per 1472 ticket. CIC needs to reject any such requests from IPV where context is not bank_account as we are supporting only this type for now.

Reply:
Ah I see thanks for explaining!